getusermedia.html

```diff
@@ -2746,14 +2763,7 @@ <h3>{{MediaDevices}}</h3>
           or removed at the same time, e.g. a camera with a microphone.</p>
         </li>
       </ol>
-      <p>Additionally, when new media input and/or output devices are made
-      available, or any available input and/or output device becomes unavailable,
-      or the system default for input and/or output devices changed, the User Agent MUST
-      run the [=device change notification steps=] in browsing contexts for
-      which [=device enumeration can proceed=] is <code>true</code> and
-      [=device information can be exposed=] is <code>true</code>, but in no
-      other contexts.</p>
-      <p>In both cases above, if a browsing context that was traversed comes
+      <p>Additionally, if a {{MediaDevices}} object that was traversed comes
       to meet the [=device enumeration can proceed=] criteria later (e.g.
       <a data-cite="!HTML/#gains-focus">gains focus</a>), the User Agent MUST
       execute the [=device change notification steps=] on the browsing
```

Comment:
For consistency with the changes above (beware dangling "context" on next line, github won't let me comment or suggest there) 
```suggestion
      execute the [=device change notification steps=] on the {{MediaDevices}} object
```

Reply:
Thank you for catching that.
[Done](https://github.com/w3c/mediacapture-main/pull/835/commits/e6008b37a5cc88962f9cc62b412014dacb5cca38) with removal of "context".